__tests__/dates.test.ts

```diff
@@ -3,7 +3,7 @@ import { settingsDateFormat } from "../src/dates";
 describe("settingsDateFormat", () => {
   it("makes a string kebob case", () => {
```

Comment:
Should this be changed since they aren't all kebab case?

Reply:
my bad, copy pasta